### PR TITLE
Rename `devtools_app_shared`'s `Split` to `SplitPane`

### PR DESCRIPTION
**Pull request by guidezpl**
This renaming will allow https://github.com/flutter/flutter/pull/143130 to land. `devtools_app_shared` is bumped to `0.0.11`.

## Pre-launch Checklist

- [x] I read the [Contributor Guide] and followed the process outlined there for submitting PRs.
- [x] I read the [Tree Hygiene] wiki page, which explains my responsibilities.
- [x] I read the [Flutter Style Guide] _recently_, and have followed its advice.
- [x] I signed the [CLA].
- [x] I listed at least one issue that this PR fixes in the description above.
- [x] I updated/added relevant documentation (doc comments with `///`).
- [x] I added new tests to check the change I am making, or there is a reason for not adding tests.

![build.yaml badge]

If you need help, consider asking for help on [Discord].

[Contributor Guide]: https://github.com/flutter/devtools/blob/master/CONTRIBUTING.md
[Tree Hygiene]: https://github.com/flutter/flutter/wiki/Tree-hygiene
[Flutter Style Guide]: https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo
[CLA]: https://cla.developers.google.com/
[Discord]: https://github.com/flutter/flutter/wiki/Chat
[build.yaml badge]: https://github.com/flutter/devtools/actions/workflows/build.yaml/badge.svg

### CODE REVIEW

File: packages/devtools_app_shared/lib/src/ui/split_pane.dart [hunk 1]
```diff
@@ -20,9 +20,9 @@ import '../utils/utils.dart';
 ///
 /// [minSizes] defines the minimum size that each child can be set to when
 /// adjusting the sizes of the children.
-final class Split extends StatefulWidget {
+final class SplitPane extends StatefulWidget {
```
Comment (kenzieschmoll) [inline packages/devtools_app_shared/lib/src/ui/split_pane.dart:23]: to make this change non-breaking, can you add another widget `Split` that returns `SplitPane` in its build method. And then mark `Split` as deprecated?

Comment (kenzieschmoll) [reply packages/devtools_app_shared/lib/src/ui/split_pane.dart:23]: Hmm actually I see now how many places we'd have to add `hide Split` to the flutter import. Can you instead change the version number to 1.0.0? And then devtools_extensions/pubspec.yaml will need to bump its devtools_app_shared dep to 1.0.0, and also it should be bumped to 0.0.15-wip.

neither package need to be published for this to unblock the flutter PR though.

Comment (kenzieschmoll) [reply packages/devtools_app_shared/lib/src/ui/split_pane.dart:23]: Great we can do `0.1.0` then for `devtools_app_shared`.

To clarify what I meant:

devtools_extensions/pubspec.yaml
```yaml
version: 0.0.15-wip
...
dependencies:
  devtools_app_shared: ^0.1.0
...
```
devtools_extensions/CHANGELOG.md
```
0.0.15-wip
* Bump `devtools_app_shared` to ^0.1.0
```

Comment (guidezpl) [reply packages/devtools_app_shared/lib/src/ui/split_pane.dart:23]: Got it, thanks. LMK if this looks good

File: packages/devtools_app/pubspec.yaml [hunk 2]
```diff
@@ -24,7 +24,7 @@ dependencies:
   collection: ^1.15.0
   dap: ^1.1.0
   dds_service_extensions: ^1.6.0
-  devtools_app_shared: ^0.0.7
+  devtools_app_shared: ^0.0.11
```
Comment (kenzieschmoll) [inline packages/devtools_app/pubspec.yaml:27]: 0.1.0

**Review by kenzieschmoll — APPROVED** [review]
lgtm with one pubspec change

File: packages/devtools_app/pubspec.yaml (hunk 2 above)
Comment (guidezpl) [reply packages/devtools_app/pubspec.yaml:27]: Done, thanks

**Comment by kenzieschmoll** [comment]
If you run `devtools_tool fix-goldens --run-id=8195797648` you can download the new golden images to your local branch. If the golden updates look good, feel free to commit them to this PR. Since we run off of bleeding edge flutter, we deal with some golden churn at times.

**Pull request merged**